node-module-system/a-wrapped-module.js

```diff
@@ -0,0 +1,5 @@
+console.log('type of exports: ', typeof exports);
+console.log('type of require: ', typeof require);
+console.log('type of module: ', typeof module);
+console.log('type of __filename: ', typeof __filename);
+console.log('type of __dirname: ', typeof __dirname);
```

Comment:
You're missing some things:

---

![image](https://user-images.githubusercontent.com/16143771/156701807-960a2b61-7365-45a9-9d20-fa8548bcdb71.png)

---

![image](https://user-images.githubusercontent.com/16143771/156701745-b49cd6bb-c1f5-4f93-9fc8-8f0fd6281236.png)

Reply:
![Screenshot (196)](https://user-images.githubusercontent.com/51901536/156734777-5c7e0b43-7499-45df-b927-acada54b4810.png)
![Screenshot (195)](https://user-images.githubusercontent.com/51901536/156734813-d93e3479-5b3b-40de-807d-010f195507cb.png)